Library/Homebrew/cask/artifact/moved.rb

```diff
@@ -94,8 +94,10 @@ def move(adopt: false, force: false, verbose: false, predecessor: nil, reinstall
           if target.writable?
             source.children.each { |child| FileUtils.move(child, target + child.basename) }
           else
-            command.run!("/bin/cp", args: ["-pR", "#{source}/*", "#{source}/.*", "#{target}/"],
-                                    sudo: true)
+            source.children.each do |child|
+              command.run!("/bin/cp", args: ["-pR", child, target + child.basename],
```

Comment:
```suggestion
              command.run!("/bin/cp", args: ["-pR", child, target/child.basename],
```

Reply:
Please also change this above while you're at it.

Reply:
Done.